api/server.go

```diff
@@ -19,16 +19,14 @@ import (
 var (
 	// Org will carry throughout the api and get tagged on resources
 	Org string
-
-	// ResultsCache is a map of in-memory caches
-	ResultsCache = make(map[string]*cache.Cache)
 )
 
 type server struct {
 	router               *mux.Router
 	version              common.Version
 	context              context.Context
 	costExplorerServices map[string]costexplorer.CostExplorer
+	resultCache          map[string]*cache.Cache
```

Comment:
What's the advantage of moving this inside `server`?

Reply:
It seems better have the cache be on the server struct and scoped with the server struct (instead of a global variable).